core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -1979,6 +1982,23 @@ protected function getAllowedExtensionsArray($properties = [])
         return $allowedExtensions;
     }
 
+    /**
+     * @param array $properties
+     *
+     * @return array|mixed|string
+     */
+    protected function getSkipExtensionsArray($properties = [])
+    {
+        $skipExtensions = $this->getOption('skipExtensions', $properties, '');
+        if (empty($skipExtensions)) {
+            $skipExtensions = [];
+        } else {
+            $skipExtensions = explode(',', $skipExtensions);
+        }
+
+        return $skipExtensions;
```

Comment:
```suggestion
        return !empty($skipExtensions) ? explode(',', $skipExtensions) : [];
```

And remove `if` statement.